flash_examples/predict/image_embedder.py

```diff
@@ -0,0 +1,28 @@
+import torch
+
+from flash.core.data import download_data
+from flash.vision import ImageEmbedder
+
+if __name__ == "__main__":
+
+    # 1. Download the data
+    download_data("https://pl-flash-data.s3.amazonaws.com/hymenoptera_data.zip", 'data/')
+
+    # 2. Create an ImageEmbedder with swav trained on imagenet.
+    # Check out SWAV: https://pytorch-lightning-bolts.readthedocs.io/en/latest/self_supervised_models.html#swav
+    embedder = ImageEmbedder(backbone="swav-imagenet", embedding_dim=128)
+
+    # 3. Generate an embedding from an image path.
+    embeddings = embedder.predict('data/hymenoptera_data/predict/153783656_85f9c3ac70.jpg')
+
+    # 4. Assert dimension
+    assert embeddings.shape == torch.Size((1, 128))
```

Comment:
What is the purpose of these asserts? For the end user

Reply:
It was to demonstrate the shape was properly generated. But not really useful.